lessons/projects/4.mdx

```diff
@@ -372,6 +365,12 @@ failure looks like:
 
 Here are tests we'll be adding:
 
+<ContentCallout emoji="💡" size="md" variant="info">
+In the next sections we'll be continuing to add groups of tests in new `describe` sections.
+Where do these go in the file? You have an existing section that starts with `describe('constructor', async () => {`.
+Ensure you paste the future `describe` sections immediately after where that previous section ends.
+</ContentCallout>
+
```

Comment:
comment out:: 
   // const symbol = await contract.symbol()
      expect(symbol).to.equal('Tier')
    })
  })//
  
new code here

//}) leave me here!! //

Reply:
Ok, I rewrote this section with a preamble showing where the previous `describe` group of tests ends, and where the new one should go.

This is similar to what you suggested.

I also decided not to paste in the whole `tier-nft.test.js`. All the `describe` blocks are in the lesson.